Astral Pathfinder/Game/Game.cpp

```diff
@@ -90,10 +94,16 @@ void Game::handleEvents() {
       shipManager->shipMovement(event);
       break;
     case SDL_KEYUP:
-      shipManager->shipMovement(event);
+      if(event.key.keysym.sym == SDLK_ESCAPE && clickFlag) {
```

Comment:
Hmm... if this ends up getting out of hand we prob gonna need an event handler class or something

Reply:
maybe, but I don't think it will?